### PR TITLE
docs: update release process

### PR DESCRIPTION
**Pull request by evegufy**
## Description

update release process

## Why

there were a couple of changes to the release process in the last months

## Issue

n/a

## Checklist

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: developer/Technical Documentation/Operations/Release-Process.md [hunk 1]
```diff
@@ -58,68 +59,29 @@ In the released version, the changelog is structured as following:
 * Technical Support
 * Bug Fixes
```
Comment (Phil91) [inline developer/Technical Documentation/Operations/Release-Process.md:60]: Could we add Known Knowns as well?

File: developer/Technical Documentation/Operations/Release-Process.md [hunk 2]
```diff
@@ -58,68 +59,29 @@ In the released version, the changelog is structured as following:
...
-
-If the tool identifies libraries that require further review, pre-vet the libraries regarding license compatibility and availability of sources. If the check is successful, create [IP Team Review Requests](https://github.com/eclipse/dash-licenses#automatic-ip-team-review-requests).
-
-### 5. Aggregate migrations (backend repo only)
+### 3. Aggregate migrations (backend repo only)
 
 Migrations should be **aggregated in the case of releasing a new version**, in order to not release the entire history of migrations which accumulate during the development process.
 
-Once a version has been released, migrations **mustn't be aggregated** in order to ensure upgradeability this also applies to **release candidates > RC1 and hotfixes**.
+Once a version has been released, migrations **mustn't be aggregated** in order to ensure upgradeability this also applies to **release candidates ) RC1 and hotfixes**.
```
Comment (Phil91) [inline developer/Technical Documentation/Operations/Release-Process.md:72]: is the ) wanted?

**Review by Phil91 — CHANGES_REQUESTED** [review]
Not sure where to place it and if it makes sense to include it in here, but when merging the rcX branch all existing PRs with the RC branch as the target should've been changed already. Otherwise github will automatically close the PRs because the rc branch will be deleted when merging

File: developer/Technical Documentation/Operations/Release-Process.md (hunk 2 above)
Comment (evegufy) [reply developer/Technical Documentation/Operations/Release-Process.md:72]: no, too much search & replace ;) thanks!

File: developer/Technical Documentation/Operations/Release-Process.md (hunk 1 above)
Comment (evegufy) [reply developer/Technical Documentation/Operations/Release-Process.md:60]: done

**Comment by evegufy** [comment]
> Not sure where to place it and if it makes sense to include it in here, but when merging the rcX branch all existing PRs with the RC branch as the target should've been changed already. Otherwise github will automatically close the PRs because the rc branch will be deleted when merging

added to 'RC: provide successive RC branch' step

**Review by oyo — APPROVED** [review]

**Review by Phil91 — APPROVED** [review]

**Pull request merged**